src/test/java/org/jseats/model/methods/RankMethodShould.java

```diff
@@ -0,0 +1,165 @@
+package org.jseats.model.methods;
+
+import com.google.common.collect.Collections2;
+import org.jseats.model.Candidate;
+import org.jseats.model.Result;
+import org.jseats.model.SeatAllocationException;
+import org.jseats.model.Tally;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+
+import static org.junit.Assert.assertEquals;
+
+public class RankMethodShould {
+
+	static Logger log = LoggerFactory.getLogger(RankMethodShould.class);
+
+	@Test
+	//Rank should not mind the order in the tallysheet
+	public void not_mind_the_order() throws Exception {
```

Comment:
Regarding java notation,

I try to stick to java notation on production code and test code except for
test method names. E.g,

notMindTheOrderOnTheTallySheetGivenAnEvenAmountOfCandidates vs
not_mind_the_order_on_the_tally_sheet_given_an_even_amount_of_candidates

Which one do you think is clearer?

to support my choice, see
http://programmers.stackexchange.com/questions/237561/naming-test-methods-in-java
(only answer)

also see this article on snake_case vs camelCase:
http://www.cs.kent.edu/~jmaletic/papers/ICPC2010-CamelCaseUnderScoreClouds.pdf

they conclude that novices understand better snake_case but with training
or experience the differences almost disappear

thanks in advance,